### PR TITLE
Bump citus debian Version

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+citus (9.2.4.citus-1) stable; urgency=low
+
+  * Fixes a release problem in 9.2.3
+
+ -- Burak Velioglu <velioglub@citusdata.com>  Tue, 31 Mar 2020 8:6:59 +0000
```
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Tue, 31 Mar 2020 8:6:59 +0000
```

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+citus (9.2.4.citus-1) stable; urgency=low
+
+  * Fixes a release problem in 9.2.3
+
+ -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Tue, 31 Mar 2020 8:6:59 +0000
```
Comment (hanefi) [inline debian/changelog]: broken timestamp

**Review by hanefi — DISMISSED** [review]
one minor issue

File: debian/changelog (hunk 2 above)
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Tue, 31 Mar 2020 08:06:59 +0000
```

Comment (onurctirtir) [reply debian/changelog]: address @hanefi's review

**Review by hanefi — APPROVED** [review]
:shipit:

**Pull request merged**